src/Components/Toast/toast.css

```diff
@@ -0,0 +1,9 @@
+.toast {
+    position: fixed;
+    top: 8rem;
+    right: 2rem;
+    transition: transform .6s ease-in-out;
+    animation: toast-in-right .7s;
+    gap: 1rem;
+    z-index: 99999;
```

Comment:
changes: don't give huge numbers for z-index and also use CSS variables for z-index